qiskit_nature/mappers/second_quantization/direct_mapper.py

```diff
@@ -10,17 +10,17 @@
 # copyright notice, and modified files need to carry a notice indicating
 # that they have been altered from the originals.
 
-"""The Direct Mapping interface."""
+"""The Direct Mapper."""
 
 from qiskit.opflow import PauliSumOp
 
 from qiskit_nature.operators.second_quantization.spin_op import SpinOp
 from qiskit_nature.operators.second_quantization.particle_op import ParticleOp
 
-from .qubit_mapping import QubitMapping
+from .qubit_mapper import QubitMapper
 
 
-class DirectMapping(QubitMapping):
+class DirectMapper(QubitMapper):
```

Comment:
Following the same idea as above, this one will get a `Spin(Qubit)Mapper` base class. Although we need to address which mappers will actually be needed here because `DirectMapper` and `LinearMapper` might be equivalent.

Reply:
Maybe that is a view to some earlier commit? In the code that is there now it is DirectMapper.

Reply:
I was just saying that it may turn out that we can remove `DirectMapper` because it might be equivalent to the `LinearMapper`. But that will become clear when we implement the latter :+1: (elsewhere obviously..)

Reply:
Ah ok my bad, too much going on! I read it as the name had not been updated!